pkg/gadgets/traceloop/tracer/tracer.go

```diff
@@ -607,7 +668,7 @@ func (t *Tracer) AttachContainer(container *containercollection.Container) error
 			}
 			evs, err := t.Read(container.ID)
 			if err != nil {
-				t.gadgetCtx.Logger().Debugf("error reading from container %s: %v", container.ID, err)
+				t.gadgetCtx.Logger().Errorf("error reading from container %s: %v", container.ID, err)
```

Comment:
@flyth I see that commit 9a0941298c639e4a52d1d98b9c69b1cb603f113a changed AttachContainer to immediately start a go routine to busy loop reading the buffer, making the traceloop gadget giving a stream of event instead of the flight recorder behaviour.

When I try this command:
```
$ sudo ./ig traceloop -c netem -r docker
```
It takes all the cpu (check with `top`) even without events.

Events are displayed instead of staying in the perf ring buffer until the user requests to get them by pressing ctrl-c as explained in [the traceloop documentation](https://github.com/inspektor-gadget/inspektor-gadget/blob/main/docs/gadgets/traceloop.md#with-ig).


I think it should be fixed first before additional changes.

Reply:
It seems to work for me :)